src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -408,7 +413,11 @@ public override Task OnDisconnectedAsync(HubConnectionContext connection)
             await SendGroupActionAndWaitForAck(connectionId, groupName, GroupAction.Remove);
         }
 
-        private async Task<bool> RemoveGroupAsyncCore(string connectionId, string groupName)
+        /// <summary>
+        /// This takes <see cref="HubConnectionContext"/> because we want to remove the connection from the
+        /// _connections list in OnDisconnectedAsync and still be able to remove groups with this method.
+        /// </summary>
+        private async Task<bool> RemoveGroupAsyncCore(HubConnectionContext connection, string groupName)
```

Comment:
Should this be called `TryRemoveGroupAsyncCore` ?

Reply:
I like it better as is since it maps to the public method `RemoveGroupAsync`